tests/e2e/scripts/test-android.sh

```
@@ -0,0 +1,78 @@
+fi
+
+# Start Metro
+echo "Starting Metro Bundler..."
+cd packages/app
+bun start > "$OUTPUT_DIR/metro.log" 2>&1 &
+METRO_PID=$!
+cd ../..
+
+# Wait for Metro to start
+echo "Waiting for Metro to start..."
+for i in {1..30}; do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)